src/components/m-datepicker-body/index.scss

```diff
@@ -21,25 +15,33 @@ $datepicker-calender-body-padding-top: 5px !default;
 }
 
 .m-datepicker-body__cell {
-  width: $with-one-seventh;
-  height: $with-one-seventh;
-
+  width: 33px;
+  height: 33px;
+  margin: 1px;
   padding-left: 0;
   padding-right: 0;
-
   background-color: transparent;
-
   border-radius: $datepicker-cell-border-radius;
   cursor: pointer;
   border: none;
 }
 
+.m-datepicker-body__cell:active {
+  outline: 0;
+  border: solid 2px $color-axa-blue;
```

Comment:
Cell border should also have `$variables`

Reply:
disagree / neutral to it